src/ClientLibrary/FactoryOrchestratorClient.cs

```diff
@@ -29,13 +32,17 @@ public partial class FactoryOrchestratorClient
         /// </summary>
         /// <param name="host">IP address of the device running Factory Orchestrator Service. Use IPAddress.Loopback for local device.</param>
         /// <param name="port">Port to use. Factory Orchestrator Service defaults to 45684.</param>
-        public FactoryOrchestratorClient(IPAddress host, int port = 45684)
+        /// <param name="serverIdentity">Distinguished name for the server.</param>

```

Comment:
can you please expand these xml comments to have the default values? It shows up in our documentation on github.io